### DISCUSSION TITLE
Processing without progress bar, no end time estimated and idle workers

### DISCUSSION THREAD

**Discussion by fcscoutinho** [sepinf-inc/IPED#324] [Q&A] [answered]
Machine translated:
I'm doing evidence processing and after a while the IPED is stopped ...
I checked the LOG and he is making some commits.
This is normal? (There are still some items to complete, but the processors are left with "Waiting for item ..." status)

[IPED-2020-11-22-09-55-17.log](https://github.com/sepinf-inc/IPED/files/5580168/IPED-2020-11-22-09-55-17.log)
![IPED](https://user-images.githubusercontent.com/38664286/99915201-0189bd80-2ce1-11eb-8ec5-61923e89f0d0.png)

**Comment by fcscoutinho** [comment]
Machine translated:
I am generating iso image with cdBurnerXP and UDF file systems. There is some type of incompatibility with IPED (I have email boxes larger than 4gb)

**Comment by lfcnassif** [comment, marked as answer]
This is a Sleuthkit-4.6.5 issue, it didn't finish to decode your ISO file (tsk_loaddb process should be still running, there is no message about it finishing in log), so total item count is unknown and IPED cannot estimate time to finish nor provide the processing progress. Workers are idle because sleuthkit item discovery eventually is very slow with some images.

Please try latest tsk_loaddb version to check if they fixed it: https://github.com/sleuthkit/sleuthkit/releases/tag/sleuthkit-4.10.1

Instead of generating ISO9660 mixed with UDF, you coud try UDF only ISO, so 7zip will be used to decode the image (TSK does not support UDF but 7zip does). Other option is creating an AD1 container, IPED decodes it without using TSK, or VHD/VMDK virtual disks (decoded using TSK).

AFAIK, IPED has no problem with large mailboxes, this does not seems related to mailboxes.

PS: Please write in english, other users from other countries could have the same issue.

**Comment by fcscoutinho** [comment]
Thank you Nassif!

**Comment by lfcnassif** [comment]
Please, if you are going to try some of the suggested approaches, report back, thanks.

**Comment by fcscoutinho** [comment]
I finished indexing without completing the process and generated an image of the files of the telematic data break by FTK (AD1) and started indexing again. I'll let you know as soon as I finish the process (it's 300Gb to index) ...

**Comment by lfcnassif** [comment]
If it has shown progress or has estimated time to finish, the above issue was gone.

**Comment by fcscoutinho** [comment]
Apparently all right !!!!

![IPED_2](https://user-images.githubusercontent.com/38664286/100082301-d7421800-2e26-11eb-91c8-e998c2e31b08.png)

**Comment by fcscoutinho** [comment]
I downloaded the latest changes from IPED, compiled it and it looks like it worked!

**Comment by fcscoutinho** [comment]
We are using IPED here at MPGO very successfully. Thank you Nassif.

**Comment by lfcnassif** [comment]
Please run the tsk_loaddb tool on your ISO image and report the original problem on sleuthkit project if it persists. We cannot report there as we can't reproduce or provide details about your image.
